src/app/technical-record/body/body.component.html

```diff
@@ -17,16 +24,22 @@
     </dd>
     <dd class="govuk-summary-list__value"></dd>
   </div>
+
+  <ng-container *ngIf="currentVehicleType === 'psv'">
+    <vtm-psv-body [activeRecord]="activeRecord"></vtm-psv-body>
```

Comment:
please adopt the convention of `vtm-body-psv` `vtm-body-hgv` for example. This will affects the files and selectors

Reply:
Resolving as this is changed